app/controllers/user/shares_controller.rb

```diff
@@ -31,14 +33,14 @@ def create
     @share.user = current_user
     # we currently only allow shares to Organisations
     @share.recipient_type = OrganisationUser.name
+    @share.save
```

Comment:
Does responder notice if `@share.save` fails?

Reply:
yes. Though i've changed the code because we can't generate a url for a birth record that is nill
Usually you use the model as the location, so it works it out - but the namespaces with user/org/admin means we can't